compiler/testData/codegen/box/callableReference/inlineArrayConstructors.kt

```diff
@@ -0,0 +1,11 @@
+// WITH_RUNTIME
+// IGNORE_BACKEND: JS_IR
```

Comment:
Broken for JS_IR on current master too

Reply:
cc @bashor

Reply:
I've reported https://youtrack.jetbrains.com/issue/KT-46520 so that it's not forgotten